libs/pages/logs/infra/src/lib/page-infra-logs.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect } from 'react'
+import { CardCluster, LayoutLogs } from '@console/shared/ui'
+import { useDocumentTitle } from '@console/shared/utils'
+import { AppDispatch, RootState } from '@console/store/data'
+import { shallowEqual, useDispatch, useSelector } from 'react-redux'
+import { useParams } from 'react-router'
+import { fetchClusterInfraLogs, selectClusterById } from '@console/domains/organization'
+import { ClusterLogs } from 'qovery-typescript-axios'
+import Row from './ui/row/row'
+
+export function PageInfraLogs() {
+  const { organizationId = '', clusterId = '' } = useParams()
+  const dispatch = useDispatch<AppDispatch>()
+
+  useEffect(() => {
+    const fetchServicesStatusByInterval = setInterval(() => {
+      dispatch(fetchClusterInfraLogs({ organizationId, clusterId }))
+    }, 3000)
+    dispatch(fetchClusterInfraLogs({ organizationId, clusterId }))
+    return () => clearInterval(fetchServicesStatusByInterval)
```

Comment:
Very good to clearInterval. That's something we have to check for all of our interval in the project.

Reply:
Yep, normally we don't have to forget this for other